middleware/gateway/source/manager/admin/cli.cpp

```diff
@@ -154,6 +154,7 @@ namespace casual
                            case Enum::connecting: return 10;
                            case Enum::pending: return 7;
                            case Enum::connected: return 9;
+                           case Enum::disconnecting: return 13;
```

Comment:
WIth some kind of macro thing and an enum-to-string-thing this could somehow be automated (and maybe constexpr'd) (still not related to this commit)

Reply:
This particular thing should be: 
```cpp
auto width( const model::Connection& value, const std::ostream&) const
{
    return std::ssize( description( value.runlevel)); 
}
```